templates/etc/httpd/conf.d/pulp_deb.conf.erb

```diff
@@ -0,0 +1,23 @@
+# -- split according to SSL-status
+<Location /pulp/deb/>
+  RewriteEngine On
+  RewriteCond %{HTTPS} on
+  RewriteRule (.+/pulp/deb/)(.*) /pulp/content/var/www/pub/deb/https/repos/$2 [DPI]
```

Comment:
This target path looks rather long. Are you sure this works?

Reply:
this is a verbatim copy of the corresponding file in pulp_deb, so i guess yes